src/lifecycle/created.js

```diff
@@ -24,6 +27,18 @@ function ensurePropertyDefinitions (elem, propertyFunctions) {
   }, {});
 }
 
+function callAttributeChangedForEachAttribute (elem, observedAttributes, definedAttribute) {
+  observedAttributes.forEach(function (name) {
+    const attr = elem.attributes[name];
+
+    // We don't call it for the defined attribute because that will have
+    // already called the handler via setAttribute().
+    if (attr && name !== definedAttribute) {
```

Comment:
I believe if you set an observed attribute in `created` that it'd fire the `attributedChanged` callback. I'm not sure what the behaviour for native should be here in V1. I'm unclear as to what the spec means here. Should this fire even if `attributeChanged` hasn't been called for existing properties? I'd think not, but then again you never know.